timelock-impl/src/main/java/com/palantir/atlasdb/timelock/lock/watch/LockWatchTestRuntimeConfig.java

```diff
@@ -31,11 +31,8 @@
 @JsonDeserialize(as = ImmutableLockWatchTestRuntimeConfig.class)
 @Value.Immutable
 public abstract class LockWatchTestRuntimeConfig {
-    @Value.Default
     @JsonProperty("namespace")
-    Optional<String> namespaceToWatch() {
-        return Optional.empty();
-    }
+    abstract Optional<String> namespaceToWatch();
```

Comment:
Looks like a separate drive by?

Reply:
I got a warning while running checkstyle, so cleaned it up :P